### PR TITLE
DIV-5913 - initial config for div-demo

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###

DIV-5913

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -58,6 +58,7 @@ k8s/**/common/ia/case-documents-api.yaml @hmcts/immigration-asylum
 k8s/**/common/ia/case-notifications-api.yaml @hmcts/immigration-asylum
 
 ## Divorce
+k8s/demo/common/divorce/*.yaml @hmcts/divorce
```
Comment (yarpo) [inline CODEOWNERS]: what does it do?

File: k8s/demo/common/divorce/div-demo.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  name: div-demo
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.div-fps: glob:prod-*
+    flux.weave.works/tag.div-hm: glob:prod-*
+    flux.weave.works/tag.div-demo-fees: glob:prod-*
+spec:
+  releaseName: div-demo
+  timeout: 900
```
Comment (yarpo) [inline k8s/demo/common/divorce/div-demo.yaml]: is this default value?

File: CODEOWNERS (hunk 1 above)
Comment (hosainnet) [reply CODEOWNERS]: Allows us to review our team to review all files matching `k8s/demo/common/divorce/*.yaml` without DevOps approval (in the future)

File: k8s/demo/common/divorce/div-demo.yaml (hunk 2 above)
Comment (hosainnet) [reply k8s/demo/common/divorce/div-demo.yaml]: I guess so, copied from https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/common/money-claims/cmc-demo.yaml#L28
We can tweak later if it doesn't work for us

**Pull request merged**